tests/utils/test_score.py

```diff
@@ -69,3 +81,115 @@ def test_squashed_variance():
     X = np.array([[1, 2], [3, 4]])
     model = FittedLinearModel(coef)
     assert sigmoid(1.0) == squashed_variance(model, X, X @ coef)
+
+
+@pytest.mark.parametrize(
+    "dataset_alt_seq_simple",
+    [((101, 0.3, 0.4))],
+    indirect=True,
```

Comment:
This is very opaque. Can't you forward arguments to the fixture like we do elsewhere to expose all the config in the calling test?